### ISSUE TITLE
Unknown mode for null_model

### ISSUE THREAD

**Issue by brian-j-smith** [tidymodels/parsnip#982]
An `"unknown"` value for the `mode` argument of `null_model()` produces the error below even thought the documentation states this is a possible value.

```r
parsnip::null_model(mode = "unknown")
#> Error in `parsnip::null_model()`:
#> ! `mode` should be one of: 'classification', 'regression'
```

**Comment by simonpcouch** [comment]
Thanks for bringing this up! Addressed in #983.

**Comment by brian-j-smith** [comment]
Thanks!
